### PR TITLE
Set up UI

### PR DESCRIPTION
**Pull request by mntedev**
I have completed the following tasks. :hand: 

:heavy_check_mark:  Created app.rb file with methods to list books, list persons, and list rentals by person id.
:heavy_check_mark:  Implemented create a new book,  create new person, create new rental features
:heavy_check_mark:  Created main.rb file with methods to render the menu options, to list all books, all persons and all rentals .
:heavy_check_mark: Implemented add new book, add new rental, add new person methods in main.rb file

### CODE REVIEW

File: main [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    date = gets.chomp
+
+    app.create_new_rental(date, person, book)
+  end
+
+  def list_rentals_by_id(app)
+    puts 'Enter person id:'
+    id = gets.chomp.to_i
+    app.list_rentals_by_person_id(id)
+  end
```
Comment (rotimiazeez) [inline main:82]: - Please note that when I tried listing rentals by a `person's id` I get the below error. Kindly fix it 😉
Screenshot from 2022-08-27 00-35-09

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @worashf  👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯
- Good commit messages 👍🏽
- Good console UI 👍🏽
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main (hunk 1 above)
Comment (worashf) [reply main:82]: **I made your suggestion, the output looks like this.**
image

**Comment by mntedev** [comment]
I made your suggestion, the output looks like the below.
image

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @worashf 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous reviewer
- Great communication with the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**